main_server.py

```
@@ -1000,6 +1175,838 @@ async def api_key_settings(request: Request):
+    try:
+        # 检查Steam客户端是否正在运行
+        if not steamworks.IsSteamRunning():
+            return JSONResponse({
+                "success": False,
+                "error": "Steam客户端未运行",
+                "message": "请确保Steam客户端已启动并正在运行"
+            }, status_code=503)
+        
+        # 检查用户是否已登录Steam
+        if not steamworks.Utils.IsOverlayEnabled():
+            logger.warning("Steam覆盖层未启用，可能会影响创意工坊功能")
```

Ruff (0.14.5)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)